### DISCUSSION TITLE
why memphis.dev url was redirected to superstream.ai

### DISCUSSION THREAD

**Discussion by panapol-p** [superstreamlabs/memphis#1661] [General]
I cannot read memphis documents because cannot access to memphis.dev, it's alway redirect to superstream.ai

<img width="1334" alt="image" src="https://github.com/memphisdev/memphis/assets/34530268/df8335ac-8c66-46a9-89b8-1f12bf5b6d36">

**Comment by panapol-p** [comment]
ok now i can access by using this one [https://docs.memphis.dev/ ](https://docs.memphis.dev/)

**Comment by Cave-Johnson** [comment]
See latest update on the Memphis project from @yanivbh1 on discord

> Hey @everyone ,
> In the last few months, our team has been rethinking the roadmap for the company and the future of Memphis Broker as a product.
> Due to different reasons, we decided to keep Memphis Broker as an OSS project only, without official commercial support, and by shifting focus to our new product, Superstream, we cannot dedicate the same amount of effort to developing Memphis Broker as we did before.
> We will “clean” the OSS version from cloud/commercial ads in the coming weeks.
> 
> Thank you, the community, and the users for using Memphis Broker. There is no better feeling for a builder than people, especially technical ones, who find your product useful, and to be honest, I might be the front-end of our team, but my part in building this amazing product is very little. It’s our amazing team that should have all the respect.

It would be nice to keep the current Memphis website as it (no redirect) for access to docs, advertising the Memphis project as it is in OSS form so that the community can still continue to work on improving Memphis
